lib/components/grpc.ts

```diff
@@ -203,6 +203,7 @@ function decodeResponse<Context extends GatewayContext>(
     ctx: Context,
     encodedFields: string[] = [],
     ErrorConstructor: AppErrorConstructor,
+    decodeAnyMessageProtoLoaderOptions?: protobufjs.IConversionOptions,
```

Comment:
May be call this option `decodeProtoLoaderOptions`?

Reply:
maybe anyMessageProtoLoaderOptions?